File: app/src/main/java/com/github/neowen/apibasedemo/media/MediaListActivity.java
package com.github.neowen.apibasedemo.media;

import com.github.neowen.apibasedemo.ApiItem;
import com.github.neowen.apibasedemo.ApiItemListActivity;

/**
 * Created by ZWX on 15/10/21.
 */
public class MediaListActivity extends ApiItemListActivity {

    @Override
    protected void addApiItemData() {
        super.addApiItemData();

        ApiItem apiItem = new ApiItem("MediaDatabase", MediaDatabaseActivity.class);
        mApiItems.add(apiItem);
    }
}

File: app/src/main/java/com/winson/apibasedemo/media/MediaListActivity.kt
package com.winson.apibasedemo.media

import com.winson.apibasedemo.base.ApiItem
import com.winson.apibasedemo.base.ApiItemListActivity
import com.winson.apibasedemo.base.BaseActivity

/**
 * @date 2020/2/22
 * @author Winson
 */
class MediaListActivity :ApiItemListActivity(){

    override fun bindData() {
        super.bindData()
        val mediaVoice = ApiItem("Media Voice")
        mediaVoice.activityClassName = MediaVoiceActivity::class.java
        items.add(mediaVoice)

        val mediaAidl = ApiItem("Media Aidl")
        mediaAidl.activityClassName = MediaAidlActivity::class.java
        items.add(mediaAidl)
    }

}